github.go

```diff
@@ -9,12 +9,15 @@ import (
 	"path"
 	"strings"
 
-	"ariga.io/gh-atlas/gen"
 	"github.com/cli/go-gh/pkg/repository"
 	"github.com/google/go-github/v49/github"
 	"golang.org/x/crypto/nacl/box"
+
+	"ariga.io/gh-atlas/gen"
 )
 
+const ACTION_FILE_PATH = ".github/workflows/ci-atlas.yaml"
```

Comment:
this const can have a smaller scope inside the `AddAtlasYAML`, consider moving it inside the function

Reply:
Also, naming-wise, I think we are using this convension?
https://stackoverflow.com/a/22689940/1708860